style.css

```diff
@@ -391,3 +429,256 @@ textarea:hover {
 #getintouch:focus {
   background-color: #a14300;
 }
+
+@media screen and (min-width: 768px) {
+  nav {
+    display: flex;
+    height: 2rem;
+  }
+
+  #logonav {
+    padding-left: 8vw;
+  }
+
+  .menuitem {
+    padding: 1vw;
+  }
+
+  #menuimg {
+    display: none;
+  }
+
+  .fullmenu {
+    display: flex;
+    cursor: pointer;
+    color: #fff;
+    align-items: center;
+    justify-content: center;
+    align-content: space-between;
+    font-family: 'Inter', sans-serif;
+    font-size: 17px;
+  }
+
+  #envelope {
+    font-size: 21px;
+    padding-right: 8vw;
+  }
+
+  .blackorange {
+    background-image: url("./img/desktop.background.png");
+    background-repeat: no-repeat;
+    background-position: right bottom;
+    background-color: #1c1a19;
+    background-size: 30vw;
+    display: flex;
+    flex-direction: row;
+    justify-content: initial;
+    align-items: center;
+  }
+
+  #texttitle {
+    font-family: 'Crete Round', serif;
+    color: #1c1a19;
+    font-size: 52px;
+    font-weight: 200;
+    padding: 25px 10px 50px 0;
+    border-bottom: #212121 1px solid;
+    margin-bottom: 50px;
+  }
+
+  .icons {
+    order: -1;
+    margin: 0 1vw 0 1vw;
+  }
+
+  #welcometext {
+    width: 40%;
+    margin: 0 1vw 0 1vw;
+  }
+
+  .iconitems {
+    display: list-item;
+  }
+
+  .cards {
+    display: grid;
+    width: 95vw;
+    grid-gap: 20px;
+    align-items: initial;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr;
+    grid-auto-flow: row;
+    padding: 30px;
+    height: auto;
+  }
+
+  .card {
+    height: auto;
+    background: linear-gradient(0deg, rgb(53, 53, 53) 0%, rgba(200, 200, 200, 0.5) 100%) url("./img/img.background.cards.png");
+    background-repeat: no-repeat;
+  }
+
+  .smoke {
+    visibility: show;
+  }
+
+  .card:hover {
+    background: url("./img/img.background.cards.png");
+    color: rgba(255, 255, 255, 0);
+    transition: 0.7s ease-in-out;
+  }
+
+  .card:hover > :nth-child(3) {
+    visibility: hidden;
+    background-color: #000;
+  }
+
+  .card .cardbottom {
+    opacity: 0;
+  }
+
+  .card:hover > .cardbottom {
+    opacity: 100;
+    transition: 0.7s ease-in-out;
+  }
+
+  .mywork {
+    width: 95vw;
+    padding: 15px;
+    height: auto;
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+  }
+
+  #imageindex {
+    min-width: 600px;
+    width: 1fr;
+    height: auto;
+    padding-right: 15px;
+  }
+
+  .aboutme {
+    background-image: url("./img/img.background.aboutme.png");
+    background-position: right;
+    background-size: 100vh;
+    justify-content: center;
+    border-bottom: #fff 1px solid;
+    width: 95vw;
+    padding: 15px;
+  }
+
+  .language {
+    display: flex;
+    height: 100vh;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: space-between;
+    width: 93.5vw;
+    padding: 25px;
+  }
+
+  .skillstitle {
```

Comment:
[OPTIONAL]

- I think it would be nice if the list of technologies are properly aligned as shown in the Figma design chosen. The technologies lists are not centered (`align-items: center`;)but rather aligned left(`align-items: flex-start). Kindly fix it. 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/178087780-5f507011-caed-4958-b9fa-84fdee3cef98.png)  | ![image](https://user-images.githubusercontent.com/63473584/178087805-394f1068-7afd-4491-8293-8722bbd1a947.png))  |